src/scripts/main.js

```diff
@@ -1,3 +1,199 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rows = [...tbody.querySelectorAll('tr')];
+let ascName = true;
+let ascPosition = true;
+let ascOffice = true;
+let ascAge = true;
+let ascSalary = true;
+const body = document.querySelector('body');
+const form = document.createElement('form');
+const notification = document.createElement('div');
+const newRow = document.createElement('tr');
+
+body.appendChild(notification);
+body.appendChild(form);
+
+form.outerHTML = `<form class="new-employee-form">
+<label>Name: <input name="name" type="text" data-qa="name" ></label>
+<label>Position: <input name="position" type="text" data-qa="position" >
+</label>
+<label>Office: <select name="office" data-qa="office">
+<option value="Tokyo">Tokyo</option>
+<option value="Singapore">Singapore</option>
+<option value="London">London</option>
+<option value="New York">New York</option>
+<option value="Edinburgh">Edinburgh</option>
+<option value="San Francisco">San Francisco</option>
+</select>
+</label>
+<label>Age: <input name="age" type="number" data-qa="age" ></label>
+<label>Salary: <input name="salary" type="number"  data-qa="salary" ></label>
+<button name="button">Save to table</button>
+</form>`;
+
+const nameInput = document.querySelector('input[name="name"]');
+const positionInput = document.querySelector('input[name="position"]');
+const officeInput = document.querySelector('select[name="office"]');
+const ageInput = document.querySelector('input[name="age"]');
+const salaryInput = document.querySelector('input[name="salary"]');
+const button = document.querySelector('button');
+
+thead.addEventListener('click', e => {
+  const columnIndex = e.target.cellIndex;
+
+  if (e.target.cellIndex === 0) {
+    if (ascName) {
+      rows.sort((a, b) => {
+        const current = a.cells[columnIndex].innerHTML;
+        const next = b.cells[columnIndex].innerHTML;
+
+        ascName = false;
+
+        return current.localeCompare(next);
+      });
+    } else {
+      rows.sort((a, b) => {
+        const current = b.cells[columnIndex].innerHTML;
```

Comment:
maybe we can extract that logic into separate function with passed parameters, like ascName, accPosition, ascOffice etc.? Seems they doing the same oprations.